src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -253,7 +253,7 @@ private function getSerializerContext(string $resourceClass, string $contextType
             $request = $this->requestStack->getCurrentRequest();
         }
 
-        if (null !== $this->serializerContextBuilder && null !== $request) {
+        if (null !== $this->serializerContextBuilder && null !== $request && !$request->attributes->get('_graphql')) {
```

Comment:
why?

Reply:
Because of https://github.com/api-platform/core/blob/master/src/Serializer/SerializerContextBuilder.php#L42. There is no serialization context in the request.

Reply:
Shouldn't there be one? I mean, usually `_api_resource_class` is declared on every route no?

Also maybe that the `SerializerContextBuilder` shouldn't throw at all.

Reply:
Maybe can we find something expliciter, but as GraphQL isn't REST, it looks okish that `_api_resource_class` doesn't exist.

Reply:
Okay then!

Reply:
should use `has` btw as pointed by @meyerbaptiste